crates/bitcoin-da/src/helpers/mod.rs

```diff
@@ -1,13 +1,51 @@
-// Tags that are used to seperate the different parts of the script
-const ROLLUP_NAME_TAG: &[u8; 1] = &[1; 1];
-const SIGNATURE_TAG: &[u8; 1] = &[2; 1];
-const PUBLICKEY_TAG: &[u8; 1] = &[3; 1];
-const RANDOM_TAG: &[u8; 1] = &[4; 1];
-const BODY_TAG: &[u8; 0] = &[];
+use core::num::NonZeroU16;
 
 #[cfg(feature = "native")]
 pub mod builders;
 pub mod compression;
 pub mod parsers;
 #[cfg(test)]
 pub mod test_utils;
+
+/// Header - first 8 bytes of any rollup transaction
+struct TransactionHeader<'a> {
+    pub(crate) rollup_name: &'a [u8],
+    pub(crate) typ: TransactionType,
```

Comment:
type*

Reply:
Type is a reserved word. I can still use it but I have to write `r#type` instead.

Reply:
that's fine, leave is as it is